src/OnboardingSPA/steps/SiteGen/Editor/Header/stylesheet.scss

```diff
@@ -160,3 +160,26 @@
 		}
 	}
 }
+
+.nfd-interface-interface-skeleton__editor {
+
+	&.editor-actions-disabled {
+
+		.nfd-editor-actions-overlay {
+			position: fixed;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			background: rgba(var(--nfd-onboarding-primary-rgb), 0.3);
+			z-index: 1;
+		}
+
+		.nfd-onboarding-header--sitegen__editor__end__save-button {
+			position: relative;
+			z-index: 2;
```

Comment:
Is there a specific reason why we have a `z-index: 2;` here? Does this button need to be on top for any particular purpose?